mapgen.lua

```diff
@@ -468,7 +523,8 @@ function vmg.generate(minp, maxp, seed)
 								data[ivm] = c_stone
 							end
 						end
-					elseif v11 + v12 > 2 ^ (y / lava_depth) and y <= lava_max_height then
+						-- Allow lava above ground to max_height, but see nodes.lua.
+					elseif y <= lava_max_height and v11 + v12 > 2^((y - lava_max_height) / lava_depth) then
```

Comment:
That's true. You would have to use the depth that you actually want lava to rise to instead of 31000. If you use a smaller value, lava will reach the surface in places, but won't be everywhere. Set to zero, things stay exactly the same as the default.

This mod isn't meant to create volcanoes. Demon_boy is still working on that. It's just an extra. Cancel the request if you don't like it. Or if you want part of it, let me know and I'll put in just the parts you like.


On Sun, 13 Sep 2015 12:28:49 -0700
Gaël Chrétien <notifications@github.com> wrote:

> With lava_max_height = 31000 to enable surface lava, most of the caves are completely filled by lava. This could be fixed by `y / lava_depth` instead of `(y - lava_max_height) / lava_depth`


-- 
Beware of he who would deny you access to information, for in his heart he dreams himself your master.
When fascism comes to America, it will be wrapped in the flag and carrying a cross.
Yes, rich people are deciding your fate. When you vote conservative, you are actively supporting that. At least when you vote liberal, you have the unions trying to pull in the opposite direction.